src/main/resources/proto/ga4gh/metadata.proto

```diff
@@ -0,0 +1,108 @@
+syntax = "proto3";
+
+package ga4gh;
+
+import "proto/google/protobuf/struct.proto";
```

Comment:
We've hit a minor issue on the server concerning the layout of the protos @calbach: ga4gh/server#876. Basically, we want to re-root to get rid of this ``proto`` prefix here. Is this possible?

Also, as an aside, would it be controversial to drop the ``main/resources`` prefix as well? Seems pretty pointless having such a deep directory structure when there's only one thing in it.